### DISCUSSION TITLE
Mojo Not Received From faucet.chia.net Now Indicates Already Sent

### DISCUSSION THREAD

**Discussion by MoonNSunM** [Chia-Network/chia-blockchain#7351] [Support]
As a user trying to start using pooling, earlier today I requested my initial faucet mojo amount from faucet.chia.net.  It is now 12 hours later.  I haven't received the initial coin required to pool and I am unable to pool at this time.  Attempted to faucet a second time.  The site now Indicates 'Already Sent'.  Please assist.
Have a great day
MoonNSunM

Receive Address = xch1gzsd4uhq8cylus3lcfh69tnvpx6nzuedplnsu2qf6tkg6hxfxars3k7zh9

**Comment by MoonNSunM** [comment]
Status update @ 24 hours ... the mojo has started to show up as pending, and I have been able to successfully create and obtain my pool key for pool plotting.  This request may be considered closed.
Have a great day all

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
